skimage/future/graph/rag.py

```diff
@@ -263,6 +266,125 @@ def rag_mean_color(image, labels, connectivity=2, mode='distance',
            http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.11.5274
 
     """
+
+
+    def define_graph(graph,labels,image, extra_arguments=[],
```

Comment:
Stylistic question.   Since `define_mean_color_rag` is not part of the public interface, essentially a "private" function used by `rag_mean_color`, should  it  be called `_define_mean_color_rag`, that is, have a leading underscore?